### DISCUSSION TITLE
Welcome to core Discussions!

### DISCUSSION THREAD

**Discussion by ajbaird** [BioGearsEngine/core#73] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by CEMoreno** [comment]
Hi. My name is Carlos Moreno. Recently I've been curious about playing with BioGears. Right now going through the challenge to make it work on Debian running on an Asus 2S SBC. Plenty of challenges. So far, no luck but I'm optimistic.

**Reply by ajbaird** [reply]
Hi Carlos! really happy that you are exploring using biogears! love chatting here but feel free to join us on our slack biogears.dev for the links to that.

In terms of the debian support, sometimes this can be a bit daunting to get running locally. I did host the arm cross compile sdk here: https://github.com/DivisionofHealthcareSimulationSciences/biogearsRelease/releases/tag/v8.1  although this is 8.1 so the current trunk is a bit farther ahead. This should be similar to the releases hosted here. Feel free to reach out to me on slack for me info, myself and steven can help walk you through getting it up and running!
